taprpc/taprootassets.proto

```diff
@@ -236,6 +236,24 @@ message AssetGroup {
     specified by the above key.
     */
     bytes asset_witness = 3;
+
+    // TODO(jhb): update to include tapscript_root
```

Comment:
Why's this need the root, wouldn't that instead be in `GroupKeyReveal` only? The witness would include the information needed to arrive at the root along with the tapscript leaf being revealed.

Reply:
True, will remove in follow-up.